test/Dapr.E2E.Test/ServiceInvocation/InvokeMethodTest.cs

```diff
@@ -0,0 +1,56 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+namespace Dapr.E2E.Test
+{
+    using System;
+    using System.Net.Http.Json;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Dapr.Client;
+    using Xunit;
+
+    public class ServiceInvocationTests : IDisposable
+    {
+        private DaprApp testApp;
+        public ServiceInvocationTests()
```

Comment:
I'd suggest accepting `ITestOutputHelper` here and passing it to `DaprApp`. This will allow you to capture all of the console output and pipe it to the test logs.

Reply:
https://xunit.net/docs/capturing-output